aggregator/types.go

```diff
@@ -33,14 +33,11 @@ import (
 	"github.com/m3db/m3x/pool"
 )
 
-// ShardFn maps a id to an aggregator shard given the total number of shards.
+// ShardFn maps a id to an aggregator shard given the total number of aggregator shards.
 type ShardFn func(id []byte, numShards int) uint32
 
-// PartitionFn maps a chunked id to a backend partition.
-type PartitionFn func(chunkedID id.ChunkedID) uint32
-
-// PartitionFnGen generates partition functions.
-type PartitionFnGen func() PartitionFn
+// AggregatedShardFn maps a chunked id of an aggregated metric to a shard.
+type AggregatedShardFn func(chunkedID id.ChunkedID, numShards int) uint32
```

Comment:
Good name, should we rename ShardFn to UnaggregatedShardFn as well?

Reply:
Meh, I think the distinction is fairly clear and `UnaggregatedShardFn` is a handful to type.